### ISSUE TITLE
the plug-in uses a R2016a onwards function

### ISSUE THREAD

**Issue by versionbayjc** [mathworks/jenkins-matlab-plugin#23]
From the documentation I understood this plug-in would work in R2013a and newer, however when I use the automatic test mode script manually in R2013a, it fails with:
```
>> runMatlabTests(true,true,false)
Undefined function 'testsuite' for input arguments of type 'char'.

Error in runMatlabTests (line 15)
suite = testsuite(pwd,'IncludeSubfolders',true);
```

And indeed if I look at the doc, "testsuite" only exists from R2016a onwards. Maybe the automatic test mode is not designed to work with releases of R2015b and older?

It seems the plug-in does recognize the MATLAB version, because it does not allow selecting Cobertura code coverage for older than R2017b.

**Comment by acampbel** [comment]
The automatic mode should indeed apply to releases older than R2016a.

**Comment by nbhoski** [comment]
Thanks @versionbayjc for bringing this to our notice. Working on it

**Comment by nbhoski** [comment]
fix is merged with number #25
